sequencer.go

```diff
@@ -55,15 +55,23 @@ func (r *SeedSequencer) Next() (seed Seed, segment IndexSegment, source SeedSegm
 	var (
 		max     uint64
 		advance = 1
+		// Limit the maximum number of chunks, in a single sequence, to avoid
+		// having jobs that are too unbalanced
+		limit = 100
```

Comment:
There are situations where this will degrade performance. Specifically when running on a filesystem that supports reflinks. There, this limit would make it slower as well as take up more disk space. I wonder if it'd be better to remove the limit parameter from the interface and instead put a limit into the nullseed, as well as the fileseed if the FS does not support cloning. The fileseed already has the `canReflink` flag which could be used to set an infinite level.
Removing the limit from the interface would be cleaner as well.

Reply:
Oh I didn't realize that.

I implemented your suggestion and now the limit is applied only if a seed can't be reflinked.